### ISSUE TITLE
myEYPD: Auto update the donut when user's manually add events

### ISSUE THREAD

**Issue by mandily-p** [BCcampus/eypd#539]
### Description 
When users manually add an event to their myEYPD page, they then need to hit an additional submit button to see a refreshed view of the page with that manually inputted data now included. 

In a polished perfect world - this would happen automagically without the need to submit any forms.

**Comment by kathreenriel** [comment]
This blocked by #264

**Comment by dshaykewich** [comment]
Won't fix.
